activities/Abecedarium.activity/play.js

```diff
@@ -26,7 +26,26 @@ enyo.kind({
 			{name: "itemCount", content: "-/-", classes: "itemCount", showing: false},		
 			{name: "back", kind: "Image", src: "images/back.png", showing: false, classes: "standardButton backButton", ontap: "backTaped"},
 			{name: "filter", kind: "Image", src: "images/filter.png", showing: false, classes: "standardButton filterButton", ontap: "filterTaped"},
-			{name: "check", kind: "Image", src: "images/check.png", showing: false, classes: "standardButton checkButton", ontap: "checkTaped"}
+			{name: "check", kind: "Image", src: "images/check.png", showing: false, classes: "standardButton checkButton", ontap: "checkTaped"},
+			{name: "gameFinished", kind: "Control", showing: false, classes: "gameFinished", 
+				components: [
+					{content: "Game Finished", classes: "gameFinsihed-message"},
+					{classes: "gameFinished-controls", components: [
+						{tag: 'span', classes: "entry replayButton", ontap: "replayTaped", 
+							components: [
+								{classes: "gameFinished-controls-text", content: "Replay"},
```

Comment:
I'm not sure a text is need

Reply:
Just to clarify. Are you talking about the `gameFinished-controls-text` class or should I just display the icons and no Replay or Select text? Also is the chosen icon for back to selection alright? Or should I change it? Could you suggest something?